src/Queue/Dev/Job.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Queue\Dev;
+
+use App\Queue;
+use SilverStripe\ORM\FieldType\DBDatetime;
+use Symbiote\QueuedJobs\Services\QueuedJob;
+
+/**
+ * Class Job
+ *
+ * This is a test job which is intended to be used for testing the queue runner
+ * the job runs multiple steps and eventually completes
+ *
+ * @property int $randomID
+ * @package App\Queue\Dev
+ */
+class Job extends Queue\Job
```

Comment:
I'd be a bit more explicit about this and call it `TestJob` (rather than relying on the task being in the `Dev` namespace). Over time, there might be other "dev jobs" with different purposes.

Reply:
This code follows the feature based namespace approach which made the code nice and portable within the bespoke code. Of course, I'm happy to change the namespace / class naming convention to whatever makes more sense within the module. This change would be done as a part of standard pull request flow.

Reply:
So if you had a second "job" that would rely on the "queue" (which all jobs do), then what would the FQCN be?